benchmarks/templates/benchmarks/tutorials/models/deepdive_2.html

```diff
@@ -140,30 +148,57 @@ <h3 class="benefits_heading is-size-3-mobile">Part 1: Defining the Model Itself<
+             However, please ensure that the outputs of any layers for which you want BrainScore to evaluate layer
+             activations are formatted as tensors. It's important to note that while PyTorch models can be created
+             such that layers output data in other formats, such as tuples or numpy arrays, BrainScore requires
+             tensor-formatted layer outputs for proper evaluation.
             </p>
         </div>
          <div class="column is-one-half">
             <img class="tutorial_dd2_2" src="{% static "/benchmarks/img/tutorials/deep_dive_2_2.png" %}" />
         </div>
     </div>
 </div>
+<div class="box leaderboard-table-component">
+    <h3 class="benefits_heading is-size-3-mobile">Part 3: How To Submit Your Completed Model</h3>
+            <p class="benefits_info is-size-5-mobile">
+                There are two ways to submit your completed model.  The first is through the <a href="https://www.brain-score.org/">BrainScore website</a>
+                and the second is directly through the <a href="https://github.com/brain-score">BrainScore Github repository</a> for
+                the appropriate domain (Vision or Language).  If you submit through the website, your submission package must be set up
+                with the proper directory structure described in Part 3 of <a href="http{% if request.is_secure %}s{% endif %}://{{ request.get_host }}/tutorials/models/deepdive_1">Deep Dive #1</a>, and
+                then compressed into a zip file.  If you submit through a Github PR, then you just need the model folder containing the four files
+                (also) described in <a href="http{% if request.is_secure %}s{% endif %}://{{ request.get_host }}/tutorials/models/deepdive_1">Deep Dive #1</a>.
+            </p>
+            <p class="benefits_info is-size-5-mobile">
+                To submit directly through Github, prepare your
+                submission folder with the four files discussed in <a href="http{% if request.is_secure %}s{% endif %}://{{ request.get_host }}/tutorials/models/deepdive_1">Deep Dive #1</a>
+                Then place this folder in <a href="https://github.com/brain-score/vision/tree/master/brainscore_vision/models">/models (for Vision)</a>
+                or <a href="https://github.com/brain-score/language/tree/main/brainscore_language/models">/models (for Language)</a>.
+                Then simply open a Pull Request and a member of the Brain-Score team
+                will review it and either merge it or suggest changes. Once this PR is merged,
+                it will be scored, and you will receive an email containing those scores.
+                NOTE: If you are submitting a plugin without other changes and don't need a review,
+                you can use the automerge tag with your PR and it will be merged automatically after merging tests.
```

Comment:
```suggestion
                you can use the automerge tag with your PR and it will be merged automatically after the unit tests pass.
```

Reply:
Kartik,
I agree with your comments and plan to make these changes.  Let's wait to merge until I make them.  I also just added Mike and he may also have a comment or two.  Thanks!!